aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -624,41 +620,28 @@ static void rsqrt_kernel(TensorIterator& iter) {
                   out_data[out_stride * (i + j)] = buffer[j];                 \
               }                                                               \
             }                                                                 \
-          },                                                                  \
-          {0, iter.numel()});                                                 \
-    });                                                                       \
-  }                                                                           \
+          }
+
+#define IMPLEMENT_FLOAT_KERNEL(op)                                                  \
+  static void op##_kernel(TensorIterator& iter) {                                   \
+    TORCH_INTERNAL_ASSERT(iter.ntensors() == 2);                                    \
+    AT_DISPATCH_FLOATING_TYPES_AND(kBFloat16, iter.dtype(), #op "_vml_cpu", [&]() { \
+      iter.serial_for_each(                                                         \
+          IMPLEMENT_ITERATOR_LAMBDA(op)                                             \
```

Comment:
```suggestion
          IMPLEMENT_ITERATOR_LAMBDA(op),                                            \
```

Reply:
Thank you, it was a stupid copy-paste error on my end.